hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.small
+9. plate, bento
+10. (\*)
+11. plate \*
+12. plate + apple
+13. bento ~ pickle
+14. plate > apple
+15. orange:first-child
+16. plate apple:only-child, plate pickle:only-child
+17. .small:last-child
+18. plate:nth-child(3)
+19. bento:nth-last-child(3)
+20. apple:first-of-type
+21. plate:nth-of-type(even)
+22. Can't solve
+23. plate apple:only-of-type
+24. apple:last-of-type, orange:last-of-type
+25. bento:empty
+26. apple:not(.small)
+27. Can't solve
+28. Can't solve
+29. Can't solve
+30. Can't solve
+31. Can't solve
+32. Can't solve
```

Comment:
Hmm, I think the hints were pretty straightforward for these, can you take a look at the hint and see if those would help ❓

Reply:
27,28,29 are similar and 30,31,32 are similar. Why don't you try going through the hints again?

Reply:
Implemented